ecs/daemon/outputs/cloudwatch_logs/task.conf

```diff
@@ -0,0 +1,32 @@
+
+[FILTER]
+    Name              ecs
+    Match             app.var.lib.docker.containers.*
+    ECS_Tag_Prefix    app.var.lib.docker.containers.
+    ADD               container    $ECSContainerName
+    ADD               ecs_task_id  $TaskID
+    ADD               ecs_family   $TaskDefinitionFamily
+
+[FILTER]
+    Name               aws
+    Match              app.var.lib.docker.containers.*
+    imds_version       v2
+    az                 false
+    ec2_instance_id    true
+
+
```

Comment:
I'll stop commenting on these blank lines after this comment.  Pretty sure they're just because this is a draft pr.